js/Storeclass.js

```diff
@@ -0,0 +1,31 @@
+// Storeclass : Handel the storage
+export default class Store {
+  // get books
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  // Add books
+  static addBooks(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] [OPTIONAL] As an optional requirement I would highly recommend to don't using methods that mutate the array such as `push`, `pop` or `splice`, instead use methods like `concert, `filter`, `slice`
    - For this change, I would ask you to take a look through your project and change these methods 🚀